src/fswAlgorithms/effectorInterfaces/torqueScheduler/torqueScheduler.h

```diff
@@ -31,10 +31,10 @@ typedef struct {
 
     /* declare these user-defined inputs */
     int    lockFlag;                               //!< flag to control the scheduler logic
-    double tSwitch;                                //!< [s] time at which controller switches to second angle
+    double tSwitch;                                //!< [s] time after epoch at which controller switches to second angle
```

Comment:
This reads wrong. Maybe you intended to write "after which the controller ..."

Reply:
I'm trying to say the switch time is the time span after the Reset() function is called.  It is not an absolute time.  I'll reword and make this clearer.